numpy/core/tests/test_umath.py

```diff
@@ -3451,6 +3471,7 @@ def test_it(self):
             assert_almost_equal(fz.real, fr, err_msg='real part %s' % f)
             assert_almost_equal(fz.imag, 0., err_msg='imag part %s' % f)
 
+    @pytest.mark.skipif(IS_WASM, reason="doesn't work")
```

Comment:
This is the first non-specific reason. Do you expect it will remain non-working? If not, maybe open an upstream issue and link it here?

Reply:
I think there are two problems that are labeled as "doesn't work":
1. `arctan` and `arctanh` fail the branch cut tests
2. `long double` precision is sometimes very bad (worse than `double` precision)

I will go back through at some point and double check so we can give more specific reasons. I'm not sure what the likelihood of fixing this is, but it isn't a very high priority for me.

Reply:
Makes sense, not super important indeed.

Reply:
Do you know what the size of `long double` in WASM is? I did a quick search, and found https://github.com/emscripten-core/emscripten/issues/5996#issuecomment-355621435 - but that's 4 years old by now.

Reply:
Yeah I am pretty darn sure that it's 128 bits.